packages/material-ui/src/InputBase/InputBase.js

```diff
@@ -87,7 +87,7 @@ export const styles = (theme) => {
       border: 0,
       boxSizing: 'content-box',
       background: 'none',
-      height: '1.1875em', // Reset (19px), match the native input line-height
+      height: '1.1885em', // Reset (19px), match the native input line-height
```

Comment:
Would it be enough? It seems to on my local tests.

```suggestion
      height: '1.1876em', // Reset (19px), match the native input line-height
```

Reply:
Yes, you're right, it would be enough `1.1876em`